### DISCUSSION TITLE
No Voltage entity

### DISCUSSION THREAD

**Discussion by minimmi** [Koenkk/zigbee2mqtt#14276] [Devices]
Hello,

I use three TS011F sockets. They also work without problems so far. Within zigbee2mqtt I also have all the energy entities. Only in Home Assistant itself I can not add the Volt entity. What can be the reason for this? Am I doing something wrong or do I have to set something somewhere? In the screenshot you can see that the volt measurement within zigbee2mqtt works without problems. 

![bild1](https://user-images.githubusercontent.com/101018377/193427850-d4ab9408-90ea-411b-b70b-a83af9ac68a1.JPG)

in the following you can see that volt is not available within HA. 

![bild2](https://user-images.githubusercontent.com/101018377/193427908-30b5f40c-14fe-40ff-b3e5-56747877b4fb.JPG)
![bild3](https://user-images.githubusercontent.com/101018377/193427913-c834b4d9-a804-4985-8296-0de32f83eb88.JPG)

**Comment by Koenkk** [comment]
You need to click on the entity in HA and enable it. It is not enabled by default to prevent too much sensor data in HA (blowing up the HA database)

**Reply by minimmi** [reply]
Thanks for your reply. Just to be sure, you mean here:
![Bild4](https://user-images.githubusercontent.com/101018377/193894821-2b421817-f2b6-47f1-b8b4-63347af756f8.jpg)

**Reply by Koenkk** [reply]
yes

**Reply by minimmi** [reply]
Thanks a lot for your help! I am still a beginner in this field and have a lot to learn :(
